### ISSUE TITLE
Multiline support problem: height not working

### ISSUE THREAD

**Issue by hosembafer** [n4kz/react-native-material-textfield#27]

**Comment by n4kz** [comment]
Please, provide more details and steps to reproduce this issue.

**Comment by hosembafer** [comment]
Try to type the code like below and will be see the difference

```
<TextField multiline={true} height={120} />

<TextInput multiline={true} height={120} />
```

**Comment by hosembafer** [comment]
An illustration and code of problem below.

```jsx
<View style={{flex: 1, flexDirection: "row", justifyContent: "center", alignItems: "center"}}>
    <View style={{width: 170}}>
        <TextInput
            label='TextInput'
            value='TextInput'
            multiline={true}
            height={200}
        />
    </View>
    <View style={{width: 170}}>
        <TextField
            label='TextField'
            value='TextField'
            multiline={true}
            height={200}
        />
    </View>
</View>
```

![input-multiline-height-broken](https://user-images.githubusercontent.com/5071289/28578342-88c816a6-716a-11e7-965c-9af586fdb806.png)

**Comment by n4kz** [comment]
Thanks for detailed explanation! Released fix in `0.8.0`.
